sorts/strand_sort.py

```diff
@@ -0,0 +1,60 @@
+from functools import partial
+import operator
+
+
+def strand_sort(arr: list, solution: list, _operator: callable):
```

Comment:
This sort would be easier to use if it returned a list instead of requiring the caller to allocate and pass in a `solution` list.

Reply:
It would also be cool if `_operator` was an optional parameter with the default set to `operator.gt`.

Reply:
@cclauss Could we move review to reopened PR: https://github.com/TheAlgorithms/Python/pull/1982? This PR was made by mistake.